src/sixty_four.rs

```diff
@@ -1,5 +1,5 @@
-use crate::TransmutingByteSlices;
-use core::{cmp, hash::Hasher};
+use crate::{TransmutingByteSlices, UnalignedBuffer};
```

Comment:
Why do we need to keep `TransmutingByteSlices` at all?

Reply:
It is still useful for the Xxhash-owned buffer (for buffering incomplete input). That buffer is aligned.

It's a trade-off between simpler code and performance on some platforms. I don't have ARM to benchmark it at hand. If you prefer simpler code I'm happy to remove TransmutingByteSlices.

Reply:
> It might still be useful for the Xxhash-owned buffer (for buffering incomplete input). That buffer is aligned.

See #37 — using `align_to` is basically wrong, anyway, so we might as well go all-in with `UnalignedBuffer`